src/components/React/Howto.js

```diff
@@ -29,117 +29,133 @@ export default function Howto(props) {
           <h2 id="user-guide" className="mb-4">
             User Guide
           </h2>
-          <Row>
-            <p>
-              Welcome to the Web-VR Volume viewer. This short guide is meant to
-              help new users on to use the application and and showcase the
-              potential its potential for future research.
-            </p>
-            <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/guide12.png"
-              alt="Example screenshot"
-              width="100%"
-            />
+          <Row class>
+            <div className="text-center">
+              <p>
+                Welcome to the Web-VR Volume viewer. This short guide is meant
+                to help new users on to navigate and analyze the 3D volume data.
+              </p>
+              <img
+                src="assets/images/howto/ridc-intro-guide.png"
+                alt="Example screenshot"
+                width="80%"
+              />
+            </div>
+
             <hr className="my-4" />
           </Row>
           <Row>
-            <p>
-              Click on the buttons to display data on the visualizer. Click on
-              the same button to remove the data from the scene. You can switch
-              between datasets by clicking the respective button.
-            </p>
-            <img
-              src="https://raw.githubusercontent.com/brown-ccv/react-aframe-volume-renderer/master/imgs/datasetbuttons.gif"
-              alt="Instructional gif on displaying data"
-              width="100%"
-            />
+            <div className="text-center">
```

Comment:
I don't think the text should be centered. If you add the padding Mary talked about I think it'll look a lot better left aligned,you'll have to delete the `className` for each div

> can you try adding `className="px-4"` to the `ModalBody` on line 28? 


```suggestion
            <div>
```

Reply:
I just did it , but it didnt change much. Please check again.

Reply:
On line 28:
```js
<ModalBody className="px-4">
```

And then fir each div (lines 33, 48, 63,, 77, 97, 113, 131)
```suggestion
            <div>
```


I would also change the images to have `width="100%` once you've done the other alignment stuff one. It won't go right to the edge because of the padding